.github/workflows/test-docker-image.yml

```diff
@@ -0,0 +1,30 @@
+name: Django CI
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build:
```

Comment:
Could you change the name of the job to test

Reply:
Done